### PR TITLE
Clean up set-gen example

### PR DESCRIPTION
**Pull request by lavalamp**
...and actually use the output.

(This is not the highest priority task I have, but it's the only one that fit in the able-to-code time I was able to scrounge up.)

@wojtek-t @thockin 

(@thockin, I have not forgotten your comments on the library. Just haven't had time.)

### CODE REVIEW

File: cmd/libs/go2idl/set-gen/sample_output/k8s.io/kubernetes/pkg/util/sets/byte.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
+/*
```
Comment (wojtek-t) [inline cmd/libs/go2idl/set-gen/sample_output/k8s.io/kubernetes/pkg/util/sets/byte.go]: To be honest - I think that we want to remove the whole second commit form this PR.
You already have third commit, where you generate something useful, so I vote for removing the whole second commit.

**Comment by wojtek-t** [comment]
Looks pretty good - thanks!
I left a bunch of comment (mostly in the first commit), but those are quite minor ones.

**Comment by wojtek-t** [comment]
@lavalamp - I've added few more comments in the individual commits, but all of them are minor.
This is very nice - thanks!

**Comment by lavalamp** [comment]
Last round resolved, I hope :)

**Comment by wojtek-t** [comment]
LGTM - thanks!

**Pull request merged**